src/test/java/com/quorum/gauge/BlockSynchronization.java

```diff
@@ -396,6 +422,37 @@ private void VerifyBlockSealingOrNotViaLogs(Node nodeName, String expectedLogMsg
         assertThat(found).as(expectedLogMsg).isTrue();
     }
 
+    @Step("Besu node <nodeName> is a Validator")
+    public void verifyBesuNodeIsValidatorFromLogs(Node nodeName) {
+        String expectedLogMsg = "Creating proposed block";
+        verifyBesuValidatorOrNotViaLogs(nodeName, expectedLogMsg, true);
+    }
+
+    @Step("Besu node <nodeName> is not a Validator")
+    public void verifyBesuNodeIsNonValidatorFromLogs(Node nodeName) {
+        String expectedLogMsg = "Creating proposed block";
+        verifyBesuValidatorOrNotViaLogs(nodeName, expectedLogMsg, false);
+    }
+
+    private void verifyBesuValidatorOrNotViaLogs(Node nodeName, String expectedLogMsg, boolean logsPresent) {
```

Comment:
this method seems to be more generic than the name. it's basically checking if that log message is present or not and can be any message, not only related to check if Besu is a validator or not.

is there other generic function like this one the t already does this? as we have other places that we grep logs

Reply:
This method filters containers by checking if the container is `besu` or not. The other method where logs are grepped does something similar for `geth` node. I will push a commit which renames the method name so that it is more generic and does not say whether we are verifying if it is a validator or not

Reply:
Pushed a commit to rename `verifyBesuValidatorOrNotViaLogs()`